buildpipeline/pipelines.json

```diff
@@ -376,7 +432,8 @@
             "RuntimeIDArg": " "
           },
           "ReportingParameters": {
-            "OperatingSystem": "Windows"
+            "OperatingSystem": "Windows",
+            "Type": "build/product/"
```

Comment:
This is where you could change these to build/test/ or something, but requires a parallel change in https://github.com/dotnet/core-eng/blob/master/mission-control-config/dotnet/coreclr/viewconfiguration.json